pod/locale/fr/LC_MESSAGES/djangojs.po

```diff
@@ -179,11 +179,19 @@ msgstr "Mettez en pause pour entrer le texte du segment entre %s et %s."
 msgid "A caption cannot contain more than 80 characters."
 msgstr "Une légende / sous-titre ne peut comporter plus de 80 caractères."
 
+#: pod/completion/static/js/caption_maker.js
+msgid "Add a caption/subtitle after this one"
+msgstr "Ajouter un(e) légende/sous-titre ci-dessous"
```

Comment:
à modifier ?

`Add a caption/subtitle below` ?

Reply:
mais below what ? "after this one" est plus précis.
Ou alors on met en FR "après celui-ci" ?

Reply:
Oui alors modifier en français. C'est mieux que les 2 versions concordent selon moi